walkthroughs/publishing-walkthroughs/walkthrough.adoc

```diff
@@ -83,16 +89,21 @@ The tutorial-web-app-operator entry is displayed.
 An editor is displayed.
 
 . Find the *Spec* > *Template* > *Parameters* entries in the YAML.
-. Create or edit the `WALKTHROUGH_LOCATIONS` parameter to include the repositories that you want to use, for example:
+. Create or edit the *WALKTHROUGH_LOCATIONS* parameter to include:
++
+* the existing value for the *WALKTHROUGH_LOCATIONS* environment variable
+* a comma separated list of the repositories that you want to add
++
+For example, if the existing value includes `https://github.com/integr8ly/tutorial-web-app-walkthroughs.git#release-no` and you want to add `https://github.com/integr8ly/example-customisations.git#master`:
 +
 ----
-WALKTHROUGH_LOCATIONS: https://github.com/integr8ly/tutorial-web-app-walkthroughs.git,https://github.com/integr8ly/example-customisations.git#master
+WALKTHROUGH_LOCATIONS: https://github.com/integr8ly/tutorial-web-app-walkthroughs.git#release-no,https://github.com/integr8ly/example-customisations.git#master
```

Comment:
`master` is probably redundant here

Reply:
yes, but leaving it as example that works, (#<release-no>) is a placeholder that won't work